datasets/camvid.py

```diff
@@ -74,8 +74,8 @@ def __init__(self, root_dir, d_type, classes=None, download=True, transform=None
             lbl = np.zeros((lbl_rgb.shape[0], lbl_rgb.shape[1]), dtype=np.uint8)
 
             for label_idx, (_, mask) in enumerate(self.label_mask_dict.items()):
-                res = (lbl_rgb == mask)
-                res = (label_idx+1) * res.all(axis=2)
+                res = lbl_rgb == mask
```

Comment:
I find the new version without parentheses confusing at first sight.

Reply:
I have no opinion on this... other than removing linter complaints